CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ New features:
 
 
 Bug fixes:
-
+* Fix duplicated `Dir#ensure_open` call. (#2992, @itarato)
```

Comment:
Needs an extra newline above, i.e., no line deleted. I'll fix it.

Also this is not a bug fix. I think better no changelog entry here, it's unlikely to observable by a user.

Reply:
Thank you!